lib/hammer_cli_foreman/compute_profile.rb

```diff
@@ -0,0 +1,412 @@
+             :format => HammerCLI::Options::Normalizers::KeyValueList.new
+      option '--interface', 'INTERFACE', _('Interface parameters, should be comma separated list of values'),
+             :format => HammerCLI::Options::Normalizers::KeyValueList.new
+      option '--interface-id', 'INTERFACE_ID', _('Interface id'),
+             :format => HammerCLI::Options::Normalizers::Number.new
+      option '--volume', 'VOLUME', _('Volume parameters, should be comma separated list of values'),
+             :format => HammerCLI::Options::Normalizers::KeyValueList.new
+      option '--volume-id', 'VOLUME_ID', _('Volume id'),
+             :format => HammerCLI::Options::Normalizers::Number.new
+
+      extend_help do |h|
+        h.section _('Provider specific options') do |h|
+          ::HammerCLIForeman::ComputeProfile.compute_resources.each do |name, provider|
+            h.section name do |h|
+              if defined?(provider.compute_profile_create_help)
+                provider.compute_profile_create_help(h)
+              else
+                provider.host_create_help(h)
+              end
+            end
+          end
+        end
+      end
+
+      def validate_options
+        super
+        validator.any(:option_compute_profile_id, :option_compute_profile_name ).required
+        validator.any(:option_compute_resource_id, :option_compute_resource_name).required
+        validator.any(:option_interface, :option_volume, :option_compute_attributes).required
+
+      end
+      def request_params
+        params = HammerCLIForeman::ComputeProfile.get_params(options)
+        interface_attr =   params['compute_attribute']['vm_attrs']['interfaces_attributes'] || {}
+        volume_attr =  params['compute_attribute']['vm_attrs']['volums_attributes'] || {}
+        vm_attr =  params['compute_attribute']['vm_attrs'] || {}
+        params['id'] =  params['compute_attribute']['id']
+        params['compute_attribute']['vm_attrs'] = params['compute_attribute']['vm_attrs'].merge(vm_attr.merge(option_compute_attributes || {} ))
+
+        if option_interface
+          params['compute_attribute']['vm_attrs']['interfaces_attributes'] = interface_attr.merge({option_interface_id => option_interface})
+        end
+        if option_volume
+          unless option_volume_id
+            params['compute_attribute']['vm_attrs']['volumes_attributes'] = volume_attr.merge({option_volume_id => option_volume})
+          end
+        end
+        params
+      end
+      success_message _('Compute profile attributes updated.')
+      failure_message _('Could not update the compute profile attributes')
+
+      build_options :without => :id
+    end
+
+    # Using the Update command because adding a new interface is done by modifying existing compute_attribute
+    class AddInterface < HammerCLIForeman::UpdateCommand
+      command_name 'add-interface'
```

Comment:
What is the difference between `compute-profile update-attributes --volume <attrs>` and `compute-profile update-volume --set-values <attrs>`?
If it is the same functionality isn't it confusing for the user to have two similar ways how to achieve the same?